test/scripts/modify-plain-specs.ts

```diff
@@ -0,0 +1,41 @@
+import fs from "fs/promises";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { ALITH_ADDRESS } from "@moonwall/util";
+import { convertExponentials } from "@zombienet/utils";
+import jsonBg from "json-bigint";
+const JSONbig = jsonBg({ useNativeBigInt: true });
+
+yargs(hideBin(process.argv))
+  .usage("Usage: $0")
+  .version("2.0.0")
+  .command(
+    `process <inputPath> <outputPath>`,
+    "Overwrites a plainSpec with Alith modifications",
+    (yargs) => {
+      return yargs
+        .positional("inputPath", {
+          describe: "Input path for plainSpecFile to modify",
+          type: "string",
+        })
+        .positional("outputPath", {
+          describe: "Output path for modified file",
+          type: "string",
+        });
+    },
+    async (argv) => {
+      const plainSpec = JSONbig.parse((await fs.readFile(argv.inputPath)).toString());
```

Comment:
We already have tools to modify genesis file without relying on JSON parser (which is slow and most of the time breaks with 10gb JSON files), any reason to redo it this way ?

Reply:
We aren’t modifying 10GB genesis here, but a 4 MB plain spec generated. This is quite quick, and modular enough to be swapped out when we port moonbeam tools functionality into moonwall